### PR TITLE
https://issues.redhat.com/browse/ACM-8789 MCE 2.3.4

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8789

### CODE REVIEW

**Review by cadawson — CHANGES_REQUESTED** [review]
I don't think there were any security CVEs on this one, were there? If not, we can remove security fixes from the statement. Other than that, it looks great. Thanks!

File: clusters/release_notes/errata.adoc [hunk 1]
```diff
@@ -7,6 +7,10 @@ For {mce-short}, the Errata updates are automatically applied when released.
 
 FIPS notice: If you do not specify your own ciphers in `spec.ingress.sslCiphers`, then the `multiclusterhub-operator` provides a default list of ciphers. For 2.4, this list includes two ciphers that are _not_ FIPS approved. If you upgrade from a version 2.4.x or earlier and want FIPS compliance, remove the following two ciphers from the `multiclusterhub` resource: `ECDHE-ECDSA-CHACHA20-POLY1305` and `ECDHE-RSA-CHACHA20-POLY1305`.
 
+== Errata 2.3.4
+
+* Delivers updates to one or more of the product container images and security fixes.
```
Comment (dockerymick) [inline clusters/release_notes/errata.adoc]:
```suggestion
* Delivers updates to one or more of the product container images.
```

**Comment by dockerymick** [comment]
@cadawson thank you for your review. Will you remind me how to confirm that there were no CVEs associated to this errata? Would it be based on the Advisory?

**Comment by cadawson** [comment]
Thanks for making the updates, @dockerymick . Yes, the prod sec CVEs would be listed in the advisory (usually by Patrick). If you have any question about whether there should be prod sec CVEs attached, you can ask Patrick. Those CVEs would make it an RHSA. Only an RHSA has the "security fixes" part in the docs errata. I understand that one of these is an RHSA at the moment, so that makes sense. I don't think it will be an RHSA by tomorrow.

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**